### PR TITLE
Added progress bar for uploading supplement file.

### PR DESCRIPTION
**Pull request by aavrug**

### CODE REVIEW

File: nuancier/static/upload.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+/**
+ * HTML5 Drag and Drop uploader
+ *
+ * This file has been adapted for the needs of pagures from the work done by
+ * Noah Petherbridge in https://github.com/kirsle/flask-multi-upload/
```
Comment (ralphbean) [inline nuancier/static/upload.js]: Maybe add a note in this comment that says the file was originally from ``flask-multi-upload``, but was then copied and modified for ``pagure``, but was then again copied and modified for ``nuancier``.

**Comment by ralphbean** [comment]
Otherwise, this looks fairly straightforward to me.  @pypingou has been much more involved in both projects, so I'll defer to him to approve the PR.  Thanks @aavrug!

**Comment by pypingou** [comment]
Code wise it looks good to me, I've just wanted to test it locally before merging it :)

**Comment by aavrug** [comment]
@ralphbean I didn't modify upload.js again just taken from the pagure. In this upload.js, there are two functions one is for drag and drop and other is for uploading. I called the only function which I need.

**Comment by pypingou** [comment]
When I tried to use it I got confused by the upload bar showing up before I click on the submit button, but looking at the filesystem in between the upload bar completing and pressing  the submit button it seems the file isn't actually uploaded to the server (or at least saved on disk), so I guess we're fine :)

Thanks @aavrug this really looks quite nice :)

**Pull request merged**